bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -680,8 +680,7 @@ bool ViewsManager::tryToEnterView(ViewNum v,
       const int64_t idx = i - minRestrictionOfPendingView;
       if (restrictionsOfPendingView[idx].isNull) {
         Assert(prePrepareMsgsOfRestrictions[idx] == nullptr);
-        // TODO(GG): do we want to start from the slow path in these cases?
```

Comment:
Are you sure that we do indeed want to start in slow path? Should you at least leave the comment?

Reply:
I read the comment different. We were starting in the slow path the whole time and so this question only made sense to me if we were starting in the fast path instead. Well, I can see it the other way. I can put the comment back.

Reply:
In the force-push below, I rebased and added the comment again.